<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QDialog" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="FonLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>250</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="RegButton">
      <property name="text">
       <string>Reg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AuthButton">
      <property name="text">
       <string>Auth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ToRegButton">
      <property name="text">
       <string>ToReg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="LoginLabel">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="Loginline"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Passwordline"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="RepeatLabel">
      <property name="text">
       <string>Repeat password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Repeatline"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>10</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
